src/app/page.tsx

```diff
@@ -61,7 +81,155 @@ export default async function Home() {
         <p className="p-2">Widget size samples</p>
         <div className="grid grid-cols-[repeat(6,200px)] auto-rows-[200px] gap-8 justify-center">
           <Widget size="small" className="bg-pink-500"></Widget>
-          <Widget size="large" className="bg-purple-500"></Widget>
+          <Widget size="large" className="bg-purple-500">
+            <div>
+              {/* weather right now */}
+              <div className="pt-8 flex">
+                <p className="text-black text-4xl">
+                  {roundNumber(currentWeatherData.temperature.value)}°
+                </p>
+                <div className="flex gap-2">
+                  <p>
+                    H:{" "}
+                    {roundNumber(weatherData.dayIntervals[0].temperature.max)}°
+                  </p>
+                  <p>
+                    L:{" "}
+                    {roundNumber(weatherData.dayIntervals[0].temperature.min)}°
+                  </p>
+                </div>
+              </div>
+              {/* weather in the following hours */}
+              <div className="flex gap-4">
+                <div className="flex flex-col">
+                  <p>{getCurrentHour()}</p>
+                  <p>{roundNumber(currentWeatherData.temperature.value)}°</p>
+                </div>
+                <div className="flex flex-col">
+                  <p>{getHourFromData(weatherData.shortIntervals[0].start)}</p>
+                  <p>
+                    {roundNumber(
+                      weatherData.shortIntervals[0].temperature.value
+                    )}
+                    °
+                  </p>
+                </div>
+                <div className="flex flex-col">
+                  <p>{getHourFromData(weatherData.shortIntervals[1].start)}</p>
+                  <p>
+                    {roundNumber(
+                      weatherData.shortIntervals[1].temperature.value
+                    )}
+                    °
+                  </p>
+                </div>
+                <div className="flex flex-col">
+                  <p>{getHourFromData(weatherData.shortIntervals[2].start)}</p>
+                  <p>
+                    {roundNumber(
+                      weatherData.shortIntervals[2].temperature.value
+                    )}
+                    °
+                  </p>
+                </div>
+                <div className="flex flex-col">
+                  <p>{getHourFromData(weatherData.shortIntervals[3].start)}</p>
+                  <p>
+                    {roundNumber(
+                      weatherData.shortIntervals[3].temperature.value
+                    )}
+                    °
+                  </p>
+                </div>
```

Comment:
You could think about rewriting this with a `.map()`, because they are the same just one number bigger :)